README.md

```diff
@@ -1,2 +1,68 @@
+- [snakemake](https://snakemake.readthedocs.io/en/stable/executing/cli.html?highlight=lint#UTILITIES) --lint: (line-length 130)
+- [snakemake dry-run](https://snakemake.readthedocs.io/en/stable/executing/cli.html#useful-command-line-arguments)
+- execution test with small dataset
+- ~~execution: integration test witt complete dataset~~: planned to be implemented 
+
+The small execution tests will make sure that the pipeline actually can be executed, i.e run and generate data. The integration test with complete data set will also evaluate the generated result to make sure that results does not change, if it's not deliberately.  
+
+We also have a [development document](https://docs.google.com/document/d/1l2v1ItZBTDaI72vQPZcaQzxwVUao78XzIJvGASAAD9E/edit?usp=sharing) describing how a hydra-genetic pipeline/module should be structured, named and what a rule should contain.    
+
+## Wiki
+Please visit our [wiki](https://github.com/hydra-genetics/.github/wiki/Welcome-to-hydra-genetics) for more information about using hydra-genetics.
+
+## Repositories 
+The current repositories can be divided into the following sections.
+
+### Tools and examples
+
+#### [hydra-genetics/tools](https://github.com/hydra-genetics/tools)
+Command line interface to create new modules/pipelines or adding a new rule to a existing project. Provides libraries used to make it easier for people not used to [pandas](https://pandas.pydata.org/) to extract information from samples and units dataframes, these dataframes are generated from [units.tsv](https://github.com/hydra-genetics/prealignment/blob/develop/workflow/schemas/units.schema.yaml) and [samples.tsv](https://github.com/hydra-genetics/prealignment/blob/develop/workflow/schemas/samples.schema.yaml) files  and used as input for hydra-genetics. 
+
+#### [hydra-genetics/docker](https://github.com/hydra-genetics/docker)
+Collection of docker files with bioinformatic tools used to execute snakemake with singularity. All dockers are automatically uploaded to [dockerhub](https://hub.docker.com/u/hydragenetics) (after being merged into master)
+
+#### [hydra-genetics/profiles](https://github.com/hydra-genetics/profiles)
+Example of configuration profiles for executing Snakemake in various computing environments
+
+
+### Analysis modules
+
+#### [hydra-genetics/prealignment](https://github.com/hydra-genetics/prealignment)
+Snakemake module containing processing steps that are be performed before sequence alignment.
+
+#### [hydra-genetics/alignment](https://github.com/hydra-genetics/alignment)
+Snakemake module containing processing steps that are be performed during sequence alignment.
+
+#### [hydra-genetics/snv_indels](https://github.com/hydra-genetics/snv_indels)
+Collection of rules used to call snv and small indels
+
+#### [hydra-genetics/annotation](https://github.com/hydra-genetics/annotation)
+Collections of rule to annotate vcf files.
+
+#### [hydra-genetics/cv_sv](https://github.com/hydra-genetics/cv_sv)
```

Comment:
Wrong linkand name: should be https://github.com/hydra-genetics/cnv_sv

Reply:
```suggestion
#### [hydra-genetics/cv_sv](https://github.com/hydra-genetics/cnv_sv)
```